karma.conf.cjs

```diff
@@ -34,9 +34,17 @@ module.exports = async function(karma) {
     ];
   }
 
+  const plugins = Object.keys(require('./package').devDependencies).flatMap(
+    (packageName) => {
+      if (!packageName.startsWith('karma-')) return []
+      return [require(packageName)]
+    }
+  );
+  plugins.push(jasmineSeedReporter);
+
   karma.set({
     frameworks: ['jasmine'],
-    plugins: ['karma-*', jasmineSeedReporter],
+    plugins,
```

Comment:
I don't get why this needs to be changed, karma requires package-lock.json?

Reply:
I added a comment explaining